src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+function sortList(usersList) {
+  const sortedUsers = [...usersList.children].sort(
+    (a, b) => salToNum(a.dataset.salary) - salToNum(b.dataset.salary)
+  );
+
+  for (const user of sortedUsers) {
+    usersList.append(user);
+  }
+}
+
+function salToNum(str) {
+  return +str.replace('$', '').split(',').join('');
+}
+
+function getEmployees(usersList) {
+  return [...usersList.children].map(
```

Comment:
`[...usersList.children]` - this can be assigned to a separate variable employees and passed to both functions